components/overflow-group/overflow-group-mixin.js

```diff
@@ -129,8 +131,13 @@ export const OverflowGroupMixin = superclass => class extends LocalizeCoreElemen
 			}
 		});
 
+		const containerStyles = {
+			minHeight: this.autoShow ? 'none' : `${this._itemHeight}px`,
+			maxHeight: this.autoShow ? 'none' : `${this._itemHeight}px`
```

Comment:
Since this is a `styleMap`, could you switch the `'none'` to `false` and it just won't add the `minHeight` and `maxHeight` styles at all?  Unless you do want them to be set to `none`

Reply:
Just tried that and it looks like then it sets the values to `false` and still adds them to the div's styles, so I'm guessing false only really works for `classMap` (which I believe we wouldn't want to use in this case since the possible min/max height is a variable)

Reply:
Huh weird, yeah I figured they just worked the same because they're named similarly 🤷